mri/operators/fourier/non_cartesian.py

```diff
@@ -378,9 +385,127 @@ def adj_op(self, x):
         return img * np.sqrt(np.prod(self.Kd))
 
 
+class gpuNUFFT:
+    """  GPU implementation of N-D non uniform Fast Fourrier Transform class.
+
+    Attributes
+    ----------
+    samples: np.ndarray
+        the mask samples in the Fourier domain.
```

Comment:
Are these the locations or the values? I am asking because there is a discrepancy with the `NFFT` class. I just saw that in `NUFFT` it's a bit different, but we still should address it and make that clear.

Reply:
Will update the doc